instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Comment from './Comment';
+import moment from 'moment';
+import styled from 'styled-components';
+
+const CommentSectionWrapper = styled.div `
+    width: 100%;
+    padding: 5px 0 0 0;
+`;
+const Timestamp = styled.p `
```

Comment:
A lot of this could have been nested inside of the main container style above. Creating a bunch of styled components can quickly make even a small file like this grow far more than intended. Think about nesting styles when possible to avoid this similar to what you used to do with LESS.